components/StyledFilters.js

```diff
@@ -35,7 +35,7 @@ const FilterButton = styled(StyledButton).attrs({
  */
 const StyledFilters = ({ filters, disabled, getLabel, onChange, selected, minButtonWidth, ...flexProps }) => {
   return (
-    <Flex data-cy="filters" py={1} css={{ overflowX: 'auto' }} {...flexProps}>
+    <Flex data-cy="filters" px={1} py={1} css={{ overflowX: 'auto' }} {...flexProps}>
```

Comment:
Sounds like this is going to impact all filers, everywhere. Do we really want this?

Reply:
This was done because filters in general tend to have a small cutoff; 

![image](https://user-images.githubusercontent.com/12435965/164019467-85180159-0ef8-41a8-91de-1926aed443e2.png)